### ISSUE TITLE
Import .hdp files directly to the campaign might be preferred

### ISSUE THREAD

**Issue by sevrick** [dmdorman/hero6e-foundryvtt#1123]
I wasn't sure how to label this as an enhancement consider it as such.

TLDR: Importing .hdp files to campaign directly seems like it would be a cleaner rather than importing them through compendiums. Reasons below. 

So you can import .hdp files into a newly created compendium, which is awesome. I have been making prefabs in HD then creating a new compendium to import them into Foundry. I then move those over to the compendiums that I have stored in my module. I would then delete the newly imported compendium. This seems redundant but I have made module that holds all my Hero System Files in one nice package so that all I have to do when making a new campaign in Hero System I can just open that module and all my files for whatever era I could possibly want is right there. If I just left them as they are when imported they would only be available in that campaign. "Why don't you just reimport them you might say?" All the images of all the weapons would not carry over and I would have to go through all my weapons again and tell foundry where to find the images I have saved for each weapon.

The end result would be you click the Create Item Icon, just like it shows now when creating compendiums, you could select your .hdp file and it would import it in a folder structure to which you could move where ever you want.

There may be reasons you didn't do it this way. I just thought I would bring it to your attention. Thx again. :)

**Comment by aeauseth** [comment]
I'm not really following.  Might we nice to see what your doing via Discord.  PM me on discord and perhaps we can setup a call for you to give me a show and tell.

**Comment by sevrick** [comment]
Sure what is your discord name?

**Comment by aeauseth** [comment]
aaron3736
